src/apps/Authenticated.tsx

```diff
@@ -45,6 +47,7 @@ const Authenticated = () => {
                                 element={<NewMaterialization />}
                             />
                         </Route>
+                        <Route path="change-set" element={<ChangeSet />} />
```

Comment:
This is the first time we have a path name composed of two words. Kebab case has always been one of my URL standards, but let me know if you have another preference.

That said, it just dawned on me that the `ChangeSet` component could be more simply named `Changes`. Suppose I had the term _change set_ too heavily on the mind. Which do you prefer?

Reply:
I think we should just smash 'em together and have the path be `changeset`